### ISSUE TITLE
Result of pretrained models on MPII

### ISSUE THREAD

**Issue by taconite** [mkocabas/EpipolarPose#2]
Hi,

Thank you for providing such a great code! I have some questions in the pretrained models on MPII.

I just did a simple verification of the accuracy of pretrained models on MPII (i.e. mpii_integral.pth.tar), and the mAP is 67.3 which is far worse than the claimed 88.5 mAP. I attached the validation config I used for testing the model (I changed the postfix to .txt so that I can upload it to github; the content is the same as a .yaml file)
[valid.txt](https://github.com/mkocabas/EpipolarPose/files/2946488/valid.txt)

Can you help me figuring out where I did wrong? I am using python3.7+pytorch1.0+Ubuntu18, so I don't think it's a cudnn issue.

thank you very much!

**Comment by mkocabas** [comment]
Hi @taconite,

Thanks for pointing out the issue. It was my mistake, sorry. I fixed the configuration file, and uploaded the correct pretrained model. Take a look at my latest [commit](https://github.com/mkocabas/EpipolarPose/commit/8543d59c4b39e46f6c725217fb5dc9944a39a793)

You need to download the model again. Then, use the `experiments/mpii/valid.yaml` to evaluate. You should obtain similar results using the updated config and pretrained model.

**Comment by taconite** [comment]
Ok, it seems to produce same accuracy as in the table, thanks!

Also I think there is something wrong with experiments/mpii/train.yaml:
https://github.com/mkocabas/EpipolarPose/blob/8543d59c4b39e46f6c725217fb5dc9944a39a793/experiments/mpii/train.yaml#L21
This line should refer to the some pretained model (on imagenet? or heatmap-based model on MPII?) instead of final integral model. More specifically, can you specify what is this pretrained model for training integral model on MPII?

thanks!

**Comment by mkocabas** [comment]
We used MPII pretrained heatmap-based model to initialize the weights. I changed that line and put the model link to the readme.

I am closing this issue since it is fixed. Feel free to reopen it if you have further problems.
